includes/class-wc-stripe-helper.php

```diff
@@ -357,7 +390,7 @@ public static function get_minimum_amount() {
 	 * @param string $setting The name of the setting to get.
 	 */
 	public static function get_settings( $method = null, $setting = null ) {
-		$all_settings = null === $method ? get_option( 'woocommerce_stripe_settings', [] ) : get_option( 'woocommerce_stripe_' . $method . '_settings', [] );
+		$all_settings = null === $method ? self::get_main_stripe_settings() : get_option( 'woocommerce_stripe_' . $method . '_settings', [] );
```

Comment:
Shouldn't we replace both `get_option`?

We could have: 
```
function get_stripe_settings( $method = '' ) { ... }
self::get_stripe_settings()
self::get_stripe_settings( $method )
```

Or maybe a new `get_stripe_settings_for_payment_method( $pm )` 🤷🏼

Reply:
Good idea! I made your first suggestion and replaced all usages of both.